Makefile

```diff
@@ -8,9 +8,10 @@ BIN_DIR=bin
 
 
 all: test build
-build: 
+build:
 		$(GOBUILD) -o $(BIN_DIR) -v ./...
-test: 
+		mv ${BIN_DIR}/cli ${BIN_DIR}/rac
```

Comment:
Here we should use the simple `go install`, which installs the binary in the GO-path. Go clean can then remove the installation. The binary gets its name from the directory it is built in.
```
build: 
		$(GOBUILD)  ./...
install:
		$(GOINSTALL) ./...
clean:
		$(GOCLEAN) ./..
```

Reply:
That's really annoying because I think `cli` is a bad name for the executable and a really good name of the directory in the code. Inversely I really don't like something like `rac` as a directory name in the code but it's a good name for the executable.